packages/plugin-e2e/src/options/index.ts

```diff
@@ -0,0 +1,10 @@
+import path = require('path');
+import { Fixtures } from '@playwright/test';
+import { PluginOptions } from '../api';
+
+const options: Fixtures<{}, PluginOptions> = {
+  featureToggles: [{}, { option: true, scope: 'worker' }],
+  provisioningRootDir: [path.join(process.cwd(), 'provisioning'), { option: true, scope: 'worker' }],
+};
+
+export default options;
```

Comment:
**Nit:** is there a reason to have `options/index.ts` instead of just a simple `options.ts`? Do we imagine having more files under `options/`?

Reply:
I think it would be wise if we don't create any nested barrel files within any packages and always use named exports when re-exporting. Using nested barrel files makes it difficult to understand what is being exported and easy to export something for a test or use elsewhere that then becomes part of the public api.

**naughty**
```
// index.ts
export * from './directory';

// ./directory/index.ts
export * from './file';

```
**nice**
```
// index.ts
export {moduleA, moduleB, typeA} from './directory/file'
```